public/app/features/dashboard/components/SubMenu/SubMenu.tsx

```diff
@@ -67,6 +70,12 @@ const mapStateToProps: MapStateToProps<ConnectedProps, OwnProps, StoreState> = (
   };
 };
 
+const styles = css`
+  display: flex;
+  flex-wrap: wrap;
+  display: contents;
+`;
+
```

Comment:
Maybe change this to:
```suggestion
function getStyles(theme: GrafanaTheme2) {
 return {
  subMenu: css`
  display: flex;
  flex-wrap: wrap;
  display: contents;
`,
};
}

```